src/Employee/Employee.java

```diff
@@ -1,16 +1,20 @@
 package Employee;
 
+import java.text.NumberFormat;
+import java.util.Objects;
+
 public class Employee {
     private String fio;
     private int department;
-    private int salary;
-    int id;
+    private double salary;
+    private final int ID;
```

Comment:
```suggestion
    private final int id;
```
camelCase всегда для названия полей

Reply:
Да, на конфе уточнил этот момент. Я думал final всегда капсом, а оказалось только static final